client/lib/domains/check-inbound-transfer-status.js

```diff
@@ -6,12 +6,14 @@ export function checkInboundTransferStatus( domainName, onComplete ) {
 		return;
 	}
 
-	wpcom.undocumented().getInboundTransferStatus( domainName, function ( serverError, result ) {
-		if ( serverError ) {
-			onComplete( serverError.error );
-			return;
-		}
-
-		onComplete( null, result );
-	} );
+	wpcom.req
+		.get( {
+			path: `/domains/${ encodeURIComponent( domainName ) }/inbound-transfer-status`,
+		} )
```

Comment:
We could also simplify this one by using the string argument shorthand:

```suggestion
		.get( `/domains/${ encodeURIComponent( domainName ) }/inbound-transfer-status` )
```